components/HackathonCard/HackathonCard.tsx

```diff
@@ -1,8 +1,10 @@
 import { Div, Row, Col, Text, Button } from "atomize"
 import Link from "next/link"
 import Date from "../Date/Date"
+import {HackathonSerializer} from "types/backend"
 
-export default function HackathonCard({ hackathon }) {
+export default function HackathonCard(props : { hackathon:HackathonSerializer }) {
+    const {hackathon} = props;
```

Comment:
We can do this in the same line as well, but it's fine.

Reply:
Done, I Actually thought of doing the same but felt that it would be shorter to do it this way than defining a new interface.

Reply:
No problem.